vmdb/app/helpers/container_helper/textual_summary.rb

```diff
@@ -13,6 +13,22 @@ def textual_group_properties
   #
 
   def textual_image
-    {:label => "image", :value => @record.image}
+    {:label => "Image", :value => @record.image}
+  end
+
+  def textual_name
+    {:label => "Name", :value => @record.name}
+  end
+
+  def textual_state
+    {:label => "State", :value => @record.state}
+  end
+
+  def textual_restart_count
+    {:label => "Restart Count", :value => @record.restart_count}
+  end
+
+  def textual_container_id
+    {:label => "Container ID", :value => @record.container_id}
```

Comment:
The container_id is a unique identifier for the container.  It's stored in the DB currently under the same label. No, it's not a relationship @abonas do you want to chime in here?